### DISCUSSION TITLE
Welcome to frontend Discussions!

### DISCUSSION THREAD

**Discussion by suchov** [kottans/frontend#359] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by suchov** [comment]
@AMashoshyna what do you think about using discussion function as one of the instruments of answering questions…? 
probably too unknown?

**Reply by derevchuk** [reply]
I don't now English

**Reply by max-petruk** [reply]
Just a piece of advice, you would be better off to start learning it

**Reply by suchov** [reply]
😆

**Reply by VladOnischenko** [reply]
😆

**Reply by derevchuk** [reply]
Hello everybody
